### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Rn486**
### **Math magicians: components**

In this pull request, I have-; 

- Deleted all the boilerplate from CRA (text, images, styles).
- Created a directory called components.
- Inside components, I created a new Calculator.js file.
- In Calculator.js, I created a React component that matches the provided design
- Used class-based components.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import { VscDebugStackframeDot } from 'react-icons/vsc';
+
+const Calculator = () => (
+  <div>
+    <ReactBootstrap.Table className="table">
+      <thead>
+        <tr className="head">
+          <th colSpan={4}>0</th>
+        </tr>
+      </thead>
```
Comment (roshan-bajgain) [inline src/components/calculator.js:16]: - [ ] According to the project requirement, the Calculator.js component should be class-based. Please implement it

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,10 @@
...
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div className="App">
+    <Calculator />
+  </div>
+);
```
Comment (roshan-bajgain) [inline src/App.js:9]: - [ ]  According to the project requirement, the App.js component should be class-based. Please implement it. eg:
`
class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
`

File: src/App.css [hunk 3]
```diff
@@ -1,39 +1,17 @@
...
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-
-  to {
-    transform: rotate(360deg);
-  }
+.body .sign {
+  background-color: #ff7518;
```
Comment (roshan-bajgain) [inline src/App.css:16]: - [ ] Please implement the vertical border between the number and kindly check the font family as well.
### Figma design
figma
### Your design
your

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Rn486, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 4]
```diff
@@ -1,29 +1,15 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.PureComponent {
```
Comment (mateo951) [inline src/App.js:5]: Addressed 🚀

**Review by mateo951 — APPROVED** [review]
### Hi @Rn486, 

Well done making the changes requested by the previous reviewer 🐐
Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**